MVVM/ViewModel/SettingsManager.cs

```diff
@@ -55,6 +62,7 @@ public void OnApplySettingsPressed()
 	{
 		smTree.ApplySettings();
 		statusBar.Text = "[color=lightgreen]Settings applied![/color]";
+		OnSettingsApplied?.Invoke();
```

Comment:
```suggestion

```

Reply:
Adressed in [3775e21](https://github.com/wisniax/RoverControlApp/pull/16/commits/3775e2150e5da267c03c2f1e671f4844844b50f8)